java-squid/src/main/java/org/sonar/java/model/InternalSyntaxToken.java

```diff
@@ -37,14 +36,16 @@
 
   private final Token token;
   private List<SyntaxTrivia> trivias;
+  private int startIndex;
+  private int endIndex;
 
 
   protected InternalSyntaxToken(InternalSyntaxToken internalSyntaxToken) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Protected method '<init>(...)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedProtectedMethod)